### PR TITLE
feat: update sidebar and base theme styles

### PR DESCRIPTION
**Pull request by YounixM**
### Summary

Update theme and sidebar as per the new designs

https://www.loom.com/share/ff565c091f3f4a9780f4b86c1dc2a7e6?sid=2b320546-7dab-4c24-876c-826703e35870

### CODE REVIEW

File: frontend/src/constants/routes.ts [hunk 1]
```diff
@@ -23,15 +22,18 @@ const ROUTES = {
...
 	UN_AUTHORIZED: '/un-authorized',
 	NOT_FOUND: '/not-found',
-	LOGS: '/logs',
-	LOGS_EXPLORER: '/logs-explorer',
-	LIVE_LOGS: '/logs-explorer/live',
-	LOGS_PIPELINES: '/pipelines',
+	LOGS_BASE: '/logs',
+	LOGS: '/logs/logs-explorer',
+	OLD_LOGS_EXPLORER: '/logs/old-logs-explorer',
+	LOGS_EXPLORER: '/logs/logs-explorer',
```
Comment (Rajat-Dabade) [inline frontend/src/constants/routes.ts:34]: Duplicate routes `LOGS` and `LOGS_EXPLORER`

File: frontend/src/container/AppLayout/index.tsx [hunk 2]
```diff
@@ -196,25 +210,68 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
...
-						<ChildrenContainer>
-							{isToDisplayLayout && !renderFullScreen && <TopNav />}
-							{children}
-						</ChildrenContainer>
-					</LayoutContent>
-				</ErrorBoundary>
-			</Layout>
+			{showTrialExpiryBanner && (
+				<div className="trial-expiry-banner">
+					You are in free trial period. Your free trial will end on{' '}
```
Comment (Rajat-Dabade) [inline frontend/src/container/AppLayout/index.tsx:241]: En-locale

File: frontend/src/container/AppLayout/index.tsx [hunk 3]
```diff
@@ -196,25 +210,68 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
...
+						<span>
+							{' '}
+							Please{' '}
+							<a className="upgrade-link" onClick={handleUpgrade}>
+								upgrade
+							</a>
+							to continue using SigNoz features.
+						</span>
+					) : (
+						'Please contact your administrator for upgrading to a paid plan.'
```
Comment (Rajat-Dabade) [inline frontend/src/container/AppLayout/index.tsx:255]: EN-Locale

**Review by Rajat-Dabade — APPROVED** [review]
Looks good to me 🚀 

Testing the following flow
https://www.notion.so/signoz/New-side-bar-testing-db44217c226f41ddb42ef3625a09edc8?pvs=4

**Review by Vikrant2520 — APPROVED** [review]

**Pull request merged**